src/calculateRentalCost.js

```diff
@@ -28,7 +28,7 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  return days * 40 - ((days >= 7) ? 50 : (days >= 3) ? 20 : 0);
```

Comment:
please never use more that 1 ?: operator in one expression
it is not readable at all

Reply:
Now that I have to re-read it, feeling the pain.. Correcting